diff -urNp -x '*.orig' sblim-sfcb-1.4.9.org/Makefile.am sblim-sfcb-1.4.9/Makefile.am
--- sblim-sfcb-1.4.9.org/Makefile.am	2014-11-25 02:43:09.000000000 +0100
+++ sblim-sfcb-1.4.9/Makefile.am	2021-10-03 23:17:39.300350989 +0200
@@ -360,6 +360,7 @@ sfcbproc_SOURCES=sfcbproc.c
 
 sfcbinst2mof_SOURCES=sfcbinst2mof.c
 sfcbinst2mof_LDADD = -lsfcFileRepository -lsfcBrokerCore
+sfcbinst2mof_DEPENDENCIES=libsfcFileRepository.la libsfcBrokerCore.la
 
 sfcbtrace_SOURCES=sfcbtrace.c
 sfcbtrace_LDADD = -lsfcBrokerCore
diff -urNp -x '*.orig' sblim-sfcb-1.4.9.org/configure.ac sblim-sfcb-1.4.9/configure.ac
--- sblim-sfcb-1.4.9.org/configure.ac	2014-11-25 03:00:34.000000000 +0100
+++ sblim-sfcb-1.4.9/configure.ac	2021-10-03 23:17:39.300350989 +0200
@@ -29,7 +29,7 @@ AC_CONFIG_SRCDIR([providerDrv.c])
 #AC_DEFUN([AC_DATAROOTDIR_CHECKED])
 
 AM_INIT_AUTOMAKE
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 
 #Source files will use the local sfcCommon files if this is set.
 #Reverse this to use the shared sfcCommon library.
@@ -163,11 +163,6 @@ if test "$SYSTEMDDIR" == "" ; then
 fi
 AC_SUBST(SYSTEMDDIR)
 
-if test "$enable_debug" != "yes" ; then
-   CFLAGS="`echo $CFLAGS | sed -e 's/-g//'`"
-   CXXFLAGS="`echo $CXXFLAGS | sed -e 's/-g//'`"
-fi
-
 if test "$enable_debug" == "yes"; then
    SFCB_CPPFLAGS="-D SFCB_DEBUG $SFCB_CPPFLAGS"
 fi
