--- sblim-sfcb-1.4.9/trace.h.orig	2014-11-25 02:43:17.000000000 +0100
+++ sblim-sfcb-1.4.9/trace.h	2015-01-09 18:27:47.226287411 +0100
@@ -94,11 +94,12 @@
 #else
 #define _SFCB_TRACE_FUNCTION(n,f)
 #define _SFCB_TRACE_VAR(f)
+#define _SFCB_TRACE_VAR_PTR(v,f)
 #define _SFCB_TRACE(LEVEL,STR)
 #define _SFCB_ENTER(n,f)
 #define _SFCB_EXIT() { return; }
 #define _SFCB_RETURN(v) {\
-   return (v); }
+   return v; }
 #define _SFCB_ABORT() {\
      printf("--- %s(%d) Abrted\n",__FILE__,__LINE__); \
      abort(); }
